keyboards/cannonkeys/vida/readme.md

```diff
@@ -0,0 +1,30 @@
+# Vida Keyboard PCB
+
+Vida PCB from CannonKeys
+
+This is for the wired hotswap and solderable versions of the Vida PCB
+
+The hotswap version does not have the same layout support as the solderable versoin and does not support per-key LEDs
```

Comment:
Same as <https://github.com/qmk/qmk_firmware/pull/24227#discussion_r1700724652>

Reply:
This is a good idea - I will do this for my open PRs

Reply:
Cheers!

Reply:
spelling:
```suggestion
The hotswap version does not have the same layout support as the solderable version and does not support per-key LEDs
```